### DISCUSSION TITLE
iOS目录下PAGViewer，编译会报'webp/decode.h' file not found

### DISCUSSION THREAD

**Discussion by YuMinHui825** [Tencent/libpag#1614] [Q&A] [answered]
iOS目录下PAGViewer，编译会报'webp/decode.h' file not found。
看相关目录下没有这几个文件，原因是？要怎么解决？

**Comment by wangwanjie** [comment]
你没有执行哪个拉依赖的shell脚本

**Comment by kevingpqi123** [comment, marked as answer]
参考这个 issue： https://github.com/Tencent/libpag/issues/21
